Description: Avoid overrunning non terminated string buffer.
 Don't call strlen on something that doesn't have a terminator.
 Use the stored length instead.
Origin: upstream, http://git.ghostscript.com/?p=ghostpdl.git;h=0bfd539
Author: Robin Watts <Robin.Watts@artifex.com>
Forwarded: yes
Bug: http://bugs.ghostscript.com/show_bug.cgi?id=699809
Last-Update: 2018-10-18
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/base/gsicc_manage.c
+++ b/base/gsicc_manage.c
@@ -3010,7 +3010,7 @@
         pval->persistent = true;
     } else {
         pval->data = (byte *)pgs->icc_manager->srcgtag_profile->name;
-        pval->size = strlen((const char *)pval->data);
+        pval->size = pgs->icc_manager->srcgtag_profile->name_length;
         pval->persistent = false;
     }
 }
